q2_coordinates/tests/test_mapper.py

```diff
@@ -27,17 +27,20 @@ def test_draw_map_from_alpha_diversity_vector(self):
         coordinates.actions.draw_map(
             metadata=self.sample_md.merge(self.alpha.view(qiime2.Metadata)),
             latitude='latitude', longitude='longitude',
-            column='observed_features')
+            column='observed_features', image='StamenTerrain',
+            color_palette='rainbow', discrete=False, missing_data='error')
```

Comment:
This test fails for me with `ValueError: Invalid transform: Spherical scatter is not supported - consider using PlateCarree/RotatedPole.`, but on cartopy==0.18 so feel free to disregard this comment if it's because of the wrong version.

Reply:
this is because of the cartopy version. In v0.16 the command still works as is. I do plan to update cartopy to the newest version at a later stage :)